peachjam/views/generic_views.py

```diff
@@ -35,7 +35,7 @@ def get(self, request, *args, **kwargs):
         return super(FilteredDocumentListView, self).get(request, *args, **kwargs)
 
     def get_base_queryset(self):
-        return self.model.objects.all()
+        return self.queryset
```

Comment:
This will break for the views that don't have `queryset` set, but instead use `model`. Do what django's views do: `return self.queryset or self.model.objects` -- that will use queryset if specified, otherwise fall back to the model queryset.